src/Component/Button.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace NotificationChannels\WhatsApp\Component;
+
+abstract class Button extends Component
+{
+    protected string $sub_type = '';
```

Comment:
Should all child classes provide a value? If so, perhaps an abstract "public abstract function subType(): string" method would be better?

Reply:
According to the [docs](https://developers.facebook.com/docs/whatsapp/cloud-api/reference/messages#components-object), Buttons must provide a sub type.
<img width="806" alt="Screenshot 2023-06-05 at 13 37 42" src="https://github.com/netflie/laravel-notification-whatsapp/assets/6515216/7fbbb695-888c-4226-9b2b-b0445073248f">

Ok I'll add a getter for the subtype